__tests__/current-alias-redirects.ts

```diff
@@ -68,65 +68,52 @@ test('no redirect when requested entity has no alias', async () => {
   await expectContainsText(response, ['Applets vertauscht'])
 })
 
-describe('redirects to first course page when requested entity is a course', () => {
-  test('when no course page is trashed', async () => {
+describe('redirects for course pages', () => {
+  beforeEach(() => {
     givenUuid({
-      id: 61682,
+      id: 42,
       __typename: 'Course',
-      alias: 'course-alias',
-      pages: [
-        { alias: '/mathe/61911/%C3%BCbersicht' },
-        { alias: '/mathe/61686/negative-zahlen-im-alltag' },
-      ],
+      alias: '/math/42/a-course',
+      currentRevision: {
+        content: JSON.stringify({
+          state: {
+            pages: [
+              { id: 'page1', title: 'Foo' },
+              { id: 'page2', title: 'Bar' },
+            ],
+          },
+        }),
+      },
     })
+  })
 
-    const response = await env.fetch({
-      subdomain: 'de',
-      pathname: '/61682',
+  test('redirect first course page to course alias', async () => {
+    const response = await localTestEnvironment().fetch({
+      subdomain: 'en',
+      pathname: '/math/42/page1/xyz',
     })
 
     const target = env.createUrl({
-      subdomain: 'de',
-      pathname: '/mathe/61911/%C3%BCbersicht',
+      subdomain: 'en',
+      pathname: '/math/42/a-course',
     })
     expectToBeRedirectTo(response, target, 301)
   })
 
-  test('when first course pages are trashed or have no current revision', async () => {
-    givenUuid({
-      id: 19479,
-      __typename: 'Course',
-      alias: 'course-alias',
-      pages: [{ alias: '/mathe/20368/%C3%BCberblick' }],
+  test('redirect course page url when title was changed', async () => {
+    const response = await localTestEnvironment().fetch({
+      subdomain: 'en',
+      pathname: '/math/42/page2/xyz',
     })
 
-    const response = await env.fetch({ subdomain: 'de', pathname: '/19479' })
-
     const target = env.createUrl({
-      subdomain: 'de',
-      pathname: '/mathe/20368/%C3%BCberblick',
+      subdomain: 'en',
+      pathname: '/math/42/page2/bar',
```

Comment:
```suggestion
      pathname: '/math/42/1a2b3c4d/bar',
```
(optional) would be more realistic